op-e2e/system_test.go

```diff
@@ -1549,6 +1526,18 @@ func testFees(t *testing.T, cfg SystemConfig) {
 	validatorRewardVaultStartBalance, err := l2Seq.BalanceAt(context.Background(), predeploys.ValidatorRewardVaultAddr, big.NewInt(rpc.EarliestBlockNumber.Int64()))
 	require.Nil(t, err)
 
+	// Fee recipients after Kroma MPT
```

Comment:
I think it would be nice to have a before and after conditional block, like if !sys.RollupConfig.IsKromaMPT(header.Time), or do you have another intention?